configs.py

```diff
@@ -32,6 +32,13 @@ def get_sys_exec_root_or_drive():
 if not postgres_host:
     raise KeyError("Application requires 'POSTGRES_HOST' to run")
 
+search_user = os.environ.get('SEARCH_USER')
+search_key = os.environ.get('SEARCH_KEY')
+if not all([search_user, search_key]):
+    print("Application requires 'POSTGRES_HOST' and 'POSTGRES_KEY for search funtionality")
```

Comment:
Should probably print `Application requires 'SEARCH_USER' and 'SEARCH_KEY for search functionality`